styles.css

```diff
@@ -1,12 +1,14 @@
-#game {
+.game {
     position: absolute;
     top: 50%;
     left: 50%;
     transform: translate(-50%, -50%);
     display: grid;
     grid-template-columns: repeat(3, auto);
```

Comment:
### use cssVariables:

- [ ] add ":root" section with --board-size variable:
```
:root{
   --board-size: 3;
}
```
- [ ] update you style to something like:
`grid-template-columns: repeat(var(--board-size), auto);`
- [ ] then in you js you can do something like:
```
document.documentElement.style.setProperty('--board-size', YOUR_NEW_SIZE);
```

---
see this [post for example](https://css-tricks.com/updating-a-css-variable-with-javascript/)

Reply:
done! works like a charm..